lib/ttfunk/table/cff.rb

```diff
@@ -31,18 +31,18 @@ def tag
         TAG
       end
 
-      def encode(mapping)
+      def encode(new2_old, old2_new)
```

Comment:
I don't quite get this change. Both mapping appear to always be derivable from another. Why do we need to pass around both?

Reply:
Yes they are derivable from each other, but passing them around is much more efficient than requiring each `encode` method to re-create them.

Reply:
I see. Please document this somewhere. Either a comment in source or commit message.

Reply:
Done